.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,74 @@
+Based on the nature of your change, please use the relevant template below and modify to suit your needs.
+
+Please ensure you copy the completed "Commit Message For Review" section below into your final commit body when merging to master. This allows us to validate that the generated release notes are clear enough for consumers to follow.
+
+Don't forget that what's written here forms the upgrade guide for consumers, and upgrading can easily become a long and arduous process. To counter this, please be as descriptive as possible, explaining the reasoning behind the change. Since there are many different consumers, changes should be documented from the perspective of the style guide itself, so avoid making specific reference to the application you're working on.
+
+
+Major release template
```

Comment:
I hear what you're saying, but I'm trying to minimise the amount of messing around you have to do when opening a PR. I want to make it as simple as deleting everything above and below the relevant section, rather than feeling like documentation surgery 😄